antora-playbook.yml

```diff
@@ -8,7 +8,7 @@ content:
     branches: HEAD
     start_path: ./
   - url: https://github.com/tinymce/tinymce-docs.git
-    branches: [ tinymce/5, tinymce/6, tinymce/7 ]
+    branches: [ tinymce/3, tinymce/5, tinymce/6, tinymce/7 ]
```

Comment:
```suggestion
    branches: [ tinymce/4, tinymce/5, tinymce/6, tinymce/7 ]
```

Reply:
This is the 4 branch, it needs to pull in the 3 branch.